### PR TITLE
feat: add organization member REST API documentation

### PR DESCRIPTION
**Pull request by jfhinchcliffe**
ℹ️ Don't merge yet. This is documentation for some feature-flagged work which is yet to be released on Buildkite

We've recently done some work to uplift the REST API - exposing organization membera (PR: https://github.com/buildkite/buildkite/pull/15036)

This PR adds documentation for that feature and should only be merged once it has been released on Buildkite.

### CODE REVIEW

File: pages/apis/rest_api/organization/members.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+<tbody>
+  <tr><th><code>id</code></th><td>UUID of the user</td></tr>
+  <tr><th><code>name</code></th><td>Name of the user</tr>
+  <tr><th><code>email</code></th><td>Email of the user</td></tr>
+</tbody>
+</table>
+
+## List organization members
+
+Returns a list of an organization's members
```
Comment (mbelton-buildkite) [inline pages/apis/rest_api/organization/members.md:17]: Missing period:

```suggestion
Returns a list of an organization's members.
```

File: pages/apis/rest_api/organization/members.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+```bash
+curl --request GET \
+  --header 'Authorization: Bearer bkua_xxx' \
```
Comment (mbelton-buildkite) [inline pages/apis/rest_api/organization/members.md:21]: There's not one pattern, but I think we should go with the one on the [Overview page](https://buildkite.com/docs/apis):

```suggestion
  --header 'Authorization: Bearer $TOKEN' \
```

File: pages/apis/rest_api/organization/members.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Required scope: `read_organizations`
+
+Success response: `200 OK`
+
+## Get an organization member
+
+```bash
+curl --request GET \
+  --header 'Authorization: Bearer bkua_xxx' \
```
Comment (mbelton-buildkite) [inline pages/apis/rest_api/organization/members.md:48]: As with the previous one:

```suggestion
  --header 'Authorization: Bearer $TOKEN' \
```

File: data/nav.yml [hunk 4]
```diff
@@ -416,6 +416,11 @@
           path: "apis/rest-api"
         - name: "Limits"
           path: "apis/rest-api/limits"
+        - name: "Organization REST API"
```
Comment (mbelton-buildkite) [inline data/nav.yml:419]: Did you have any thoughts behind the position of this?

I can see it up here being product-agnostic, but think it probably gives it too much prominence. I'd suggest either adding it below the existing ones to keep the change small. Or, perhaps this is a good time to alphabetize them.

I'd suggest going with alphabetizing:

- Agent REST API
- Analytics REST API
- Pipelines REST API
- Organization REST API

**Review by mbelton-buildkite — APPROVED** [review]
I've left some comments to address, but it looks good in general. 😊

File: data/nav.yml (hunk 4 above)
Comment (jfhinchcliffe) [reply data/nav.yml:419]: Can do!

**Comment by jfhinchcliffe** [comment]
Consolidated into this PR: https://github.com/buildkite/docs/pull/2612

**Pull request closed without merging**